apps/desktop/src/lib/forge/github/githubPrService.test.ts

```diff
@@ -36,4 +37,108 @@ describe('GitHubPrService', () => {
 		const pr = await service?.fetch(123);
 		expect(pr?.title).equal(title);
 	});
+
+	test('should detect stacked PR across forks error', async () => {
+		const mockError = {
+			message: 'Validation Failed',
+			response: {
+				data: {
+					message: 'Validation Failed',
+					errors: [
+						{
+							resource: 'PullRequest',
+							field: 'base',
+							code: 'invalid'
+						}
+					]
+				}
+			}
+		};
+
+		vi.spyOn(octokit.pulls, 'create').mockRejectedValue(mockError);
+
+		try {
+			await service?.createPr({
+				title: 'Test PR',
+				body: 'Test body',
+				draft: false,
+				baseBranchName: 'feature-branch',
+				upstreamName: 'my-branch'
+			});
+			expect.fail('Should have thrown an error');
+		} catch (err: any) {
+			expect(err.code).toBe(Code.GitHubStackedPrFork);
+		}
+	});
+
+	test('should not detect stacked PR error for other validation errors', async () => {
+		const mockError = {
+			message: 'Validation Failed',
+			response: {
+				data: {
+					message: 'Validation Failed',
+					errors: [
+						{
+							resource: 'PullRequest',
+							field: 'title',
+							code: 'missing'
+						}
+					]
+				}
+			}
+		};
+
+		vi.spyOn(octokit.pulls, 'create').mockRejectedValue(mockError);
+
+		try {
+			await service?.createPr({
+				title: 'Test PR',
+				body: 'Test body',
+				draft: false,
+				baseBranchName: 'feature-branch',
+				upstreamName: 'my-branch'
+			});
+			expect.fail('Should have thrown an error');
+		} catch (err: any) {
+			expect(err.code).not.toBe(Code.GitHubStackedPrFork);
+		}
+	});
```

Comment:
Perhaps we could manually remove this test case, it feels unnecessary.

Reply:
Thanks, done.